### ISSUE TITLE
3rd Modal for creating a Drill Profile should have `Save` button repositioned for easier accessibility

### ISSUE THREAD

**Issue by codeabiswas** [codeabiswas/ball_e_gui#8]
Currently, the 3rd modal for creating a Drill Profile (in function `create_new_drill_profile_modal_page_three()` in `DrillProfilesScreen`) has the `Save` button positioned all the way in the bottom. In the small screen selected with the Jetson Nano, this is a bit too below and it is hard to click on/receive clicks on that button. It would be helpful if the button is repositioned horizontally along with the `Back` button to improve accessibility (i.e. use `QHBoxLayout()` instead of `QVBoxLayout()` to contain the buttons in the modal)
